### PR TITLE
docs: Update Docs to Reflect Change to .env.example

### LINKED ISSUE #3560: readME: Update ReadME for local installation as there is change in `.env`
**Describe the bug**

Since now the `.env.example` points to `heroku server`, Change the readME if the user wants to work on server too.

**Current**
```
By default, the .env.example file specifies the API_HOST as localhost:5000 and assumes, that the open-event-server is already up and running. If you intend to work on issues which involve both the frontend, and the backend, please install and set it up first before proceeding to run the frontend. If however, you intend to work on just the frontend, please change the URL for API_HOST to https://open-event-api-dev.herokuapp.com which is a test deployment of the open-event-server.
```

Change it according to latest

### PR DESCRIPTION
**Pull request by theaquarium**
Fixes #3560

#### Short description of what this resolves:
Updates the documentation to reflect the change to the default `.env.example` file which now points to Heroku and not localhost.

#### Changes proposed in this pull request:

- Update Local.md doc to reflect changes to default config
-
-

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: docs/installation/local.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ brew reinstall gettext
 brew link --force gettext 
 ```
 
-- By default, the `.env.example` file specifies the `API_HOST` as `localhost:5000` and assumes, that the [open-event-server](https://github.com/fossasia/open-event-server) is already up and running. If you intend to work on issues which involve both the frontend, and the backend, please install and set it up first before proceeding to run the frontend. **If however, you intend to work on just the frontend, please change the URL for `API_HOST` to `https://open-event-api-dev.herokuapp.com` which is a test deployment of the open-event-server.**
+- By default, the `.env.example` file specifies the `API_HOST` as `https://open-event-api-dev.herokuapp.com` which is a test deployment of the open-event-server. If you intend to work on just the frontend, this is sufficient. **If however, you intend to work on issues which involve both the frontend and the backend, you must have the [open-event-server](https://github.com/fossasia/open-event-server) already up and running. Please install and set it up first before changing the URL for `API_HOST` to `localhost:5000` and proceeding to run the frontent.**
```
Comment (kushthedude) [inline docs/installation/local.md]: Correct the typo in last line

**Comment by kushthedude** [comment]
Please use the `Semantic PR Naming` for naming your PR's

**Comment by theaquarium** [comment]
Sorry about that. Should be fixed.

**Comment by kushthedude** [comment]
@theaquarium Squash your commits

**Pull request merged**